lib/app/config.rb

```diff
@@ -36,6 +36,9 @@
     optional(:gitlab).hash do
       required(:api_token).value(:string)
     end
+
+    required(:use_analysis_icu).value(:bool)
+    required(:content_language_code).value(:string)
```

Comment:
Hmmmm I thought this is actually a part of connector configuration - I saw that you specify language in UI when you create a connector, or is it a different thing?

Reply:
Yeah, it's the same thing. It's needed for registering connectors via CLI, bypassing Kibana.

Reply:
On second thought, I think I'll make them optional. Since the CLI won't be the "official" way of creating connectors. So this PR is not needed. =)